aitinero/convex/events.ts

```diff
@@ -0,0 +1,98 @@
+import { query, mutation } from './_generated/server';
+import { Infer, v } from 'convex/values';
+import {
+  convertTimestampToMilliseconds,
+  filterUndefinedProperties
+} from './utils';
+import { EVENT_STATUS } from './schema';
+
+const EVENTS_TABLE = 'events';
```

Comment:
Maybe we can call this EVENTS_TABLE_NAME instead to sync with trips.ts
```suggestion
const EVENTS_TABLE_NAME = 'events';
```